core/include/kernel/interrupt.h

```diff
@@ -32,6 +32,12 @@
 
 #define ITRF_TRIGGER_LEVEL	(1 << 0)
 
+#define cpumask_of(cpu)		(1 << cpu)
```

Comment:
Yes, the CPU number here just corresponding to the CPU interface. So should we complete a map here which can ensure the right CPU when get the CPU number from get_core_pos()?